### PR TITLE
1771 build partners page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- Add `/partners` page

## QA
- Run the website locally using `./run`
- Go to [/partners](http://0.0.0.0:8002/partners) and compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1771)

## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1771

### CODE REVIEW

File: templates/partners/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    background-position: left top, right top, right 101%;
+    background-repeat: no-repeat;
+    background-size: 70% 80%, 100% 100%, 105% 30%, 100% 100%;
+    color: #fff;
+    padding-bottom: 6rem;
+    padding-top: 6rem;
+  }
+</style>
+<section class="p-strip is-deep">
+  <div class="row u-equal-heigh" style="grid-gap: .5rem 2rem;">
```
Comment (anthonydillon) [inline templates/partners/index.html]: u-equal-heigh**t**

File: templates/partners/index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/0e94b67f-Cloud.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: Missing an alt

File: templates/partners/index.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/cloud">Cloud&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Ubuntu on clouds is optimised, tested and certified by Canonical for performance and security</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/55587253-Server.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/server">Server&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Certified hardware for efficient data centers from the large-scale core to distributed edge</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/63dbd58d-Desktop.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/desktop">Desktop&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Pre-installed and certified Ubuntu on desktops, laptops and workstations</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/e82bf16b-Silicon.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/silicon">Silicon&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Enablement and optimisation of Ubuntu on a wide range of CPUs and chipsets, boards and SOCs</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/3b3e7325-OpenStack.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/openstack">OpenStack&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Third-party software extends Canonical Openstack with compliance, backup,
+            storage and SDN</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/466c2c86-Apps+%26+Snaps.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/apps&snaps">Apps & Snaps&nbsp;&rsaquo;</a></h1>
```
Comment (anthonydillon) [inline templates/partners/index.html]: please encode the `&` with `&amp;`

File: templates/partners/index.html [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+          <p class="p-media-object__content">Thousands of publishers deliver an incredible range of apps for cloud,
+            workstation and devices</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/b77d6733-Resellers.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/resellers">Resellers&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Channel partners ensure that local Ubuntu expertise is available on every
+            continent</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/4ef8b431-Devices+%26+IoT.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/devices&iot">Devices & IoT&nbsp;&rsaquo;</a></h1>
```
Comment (anthonydillon) [inline templates/partners/index.html]: can we put & in urls? I think the url should be devices-and-iot

File: templates/partners/index.html [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+          <p class="p-media-object__content">ODM, factory, integration and device management partners get your device to market faster</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/823aa481-Charms.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+        <div class="p-media-object__details">
+          <h1 class="p-media-object__title"><a href="/partners/charms">Charms&nbsp;&rsaquo;</a></h1>
+          <p class="p-media-object__content">Cloud-native ops store for public, private and container infrastructure</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 col-medium-3">
+      <div class="p-media-object--strip">
+        <img src="https://assets.ubuntu.com/v1/bee9287f-Hosting.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/index.html]: alt

File: templates/partners/index.html (hunk 1 above)
Comment (solazio) [reply templates/partners/index.html]: actually not needed

**Pull request merged**